recipes/natural_language_processing/chatbot/Makefile

```diff
@@ -4,14 +4,15 @@ APPIMAGE ?= quay.io/ai-lab/${APP}:latest
 SERVERIMAGE ?= quay.io/ai-lab/llamacpp-python:latest
 SSHPUBKEY ?= $(shell cat ${HOME}/.ssh/id_rsa.pub;)
 BOOTCIMAGE ?= quay.io/ai-lab/${APP}-bootc:latest
+FROM ?=
 
 .PHONY: build
 build:
 	podman build -f builds/Containerfile -t ${APPIMAGE} .
 
 .PHONY: bootc
 bootc: quadlet
-	podman build --cap-add SYS_ADMIN --build-arg "SSHPUBKEY=$(SSHPUBKEY)" -f bootc/Containerfile -t ${BOOTCIMAGE} .
+	podman build $(FROM) --cap-add SYS_ADMIN --build-arg "SSHPUBKEY=$(SSHPUBKEY)" -f bootc/Containerfile -t ${BOOTCIMAGE} .
```

Comment:
Here FROM just ends up being a generic argument to podman, how about doing:
```
podman build $${FROM:+--from $${FROM}}
```

SO the user types `make FROM=image` instead of `make FROM="--from image"`.